pkg/repositories/database.go

```diff
@@ -139,21 +140,24 @@ func createPostgresDbIfNotExists(ctx context.Context, gormConfig *gorm.Config, p
 	result := gormDb.Exec(createDBStatement)
 
 	if result.Error != nil {
-		return nil, result.Error
+		if !isPgErrorWithCode(result.Error, pqDbAlreadyExistsCode) {
+			logger.Warningf(ctx, "Got DB already exists for [%v], skipping...", pgConfig.DbName)
```

Comment:
nit: this log statement is wrong, right?

Reply:
is it? why?

Reply:
because you're checking that it's not pqDbAlreadyExistsCode

Reply:
🤦